### ISSUE TITLE
Publish documentation

### ISSUE THREAD

**Issue by brianhelba** [ansible-community/molecule-docker#39]
The built Sphinx documentation for the `Docker` driver is no longer being built and published anywhere.

Once this is fixed, it would also be great to point from https://molecule.readthedocs.io/en/latest/configuration.html#driver back to this project, to increase discoverability.

**Comment by bendem** [comment]
Yes please, it's kinda hard to use molecule right now, the documentation is basically all gone.
